### DISCUSSION TITLE
Order parameter problem

### DISCUSSION THREAD

**Discussion by zahidhasan83** [idaholab/moose#26430] [Q&A Modules: Phase field]
Dear MOOSE enthusiasts,
I am currently conducting a grain-growth simulation using MOOSE phase-field. The simulation runs upto a certain time steps smoothly. However, it ends with the following error message "Unable to find any suitable order parameters for remapping while working with the following grain IDs:
{164, 492}

Possible Resolutions:
        - Add more order parameters to your simulation (8 for 2D, 28 for 3D)
        - Increase adaptivity or reduce your grain boundary widths
        - Make sure you are not starting with too many grains for the mesh size"

For your kind information, I have a 2D model with 10 order parameters, and one reserved order parameter. 
Next, I have increased the researved order parameter to 8. Without success, this time the computation of Jacobian took longer, and the computation ended earlier showing killed.  

Would you be kind to let me know, what I could do to solve the problem? Your expert advices will be appreciated.

Thanks and best regards,
Zahid

**Comment by GiudGiud** [comment]
Hello

> the computation ended earlier showing killed.

are you running out of memory by any chance?
Can you try something between 1 and 8 ?
Or can you take steps to use less memory?
